MarvelApp/MarvelApp/Controllers/HomeViewController.swift

```diff
@@ -1,7 +1,81 @@
 import UIKit
 
 final class HomeViewController: UIViewController {
+
+    let sectionTitles = [
+        "Trending",
+        "Favorite",
+        "New Releases",
+        "Recomment",
+        "Popular"
+    ]
+
+    @IBOutlet private weak var homeTableView: UITableView!
+
```

Comment:
thêm enum để quản lý các constraint nhé e 
```suggestion
private enum LayoutOptions {
        static let heightDefault: CGFloat = 50
    }
```